src/commands/delifeq.json

```
@@ -0,0 +1,61 @@
+{
+    "DELIFEQ": {
+        "summary": "Delete key if value matches string.",
+        "complexity": "O(1)",
+        "group": "string",
+        "since": "8.2.0",
+        "arity": 3,
+        "function": "delifeqCommand",
+        "command_flags": [
+            "WRITE"
+        ],
+        "acl_categories": [
+            "KEYSPACE",
+            "STRING"
+        ],
+        "key_specs": [
+            {
+                "flags": [
+                    "RM",
+                    "ACCESS",
+                    "DELETE"
+                ],
+                "begin_search": {
+                    "index": {
+                        "pos": 1
+                    }
+                },
+                "find_keys": {
+                    "range": {
+                        "lastkey": -1,
```

I believe this is why you get the CROSSSLOT error in the test. This says the rest of the arguments are keys. It's actually just one key.

Compare to set.json (The SET command) and see [README.md](https://github.com/valkey-io/valkey/blob/unstable/src/commands/README.md#key-specs) in the same directory.

```suggestion
                        "lastkey": 0,
```